aws/scenarios/vm/os/fedora.go

```diff
@@ -0,0 +1,41 @@
+package os
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/DataDog/test-infra-definitions/aws"
+	"github.com/DataDog/test-infra-definitions/aws/ec2/ec2"
+	commonos "github.com/DataDog/test-infra-definitions/common/os"
+)
+
+type fedora struct {
+	*unix
+	*commonos.Unix
+	env aws.Environment
+}
+
+func newFedora(env aws.Environment) *fedora {
+	return &fedora{
+		unix: &unix{},
+		env:  env,
+		Unix: commonos.NewUnix(&env),
+	}
+}
+func (*fedora) GetSSHUser() string { return "fedora" }
+
+func (u *fedora) GetImage(arch commonos.Architecture) (string, error) {
+	switch arch {
+	case commonos.AMD64Arch:
+		return ec2.SearchAMI(u.env, "679593333241", "Fedora-Cloud-Base-34-1.2*-standard-*", string(arch))
```

Comment:
> There might something in the the AWS provider to accept terms automatically.

Probably, but I don't expect we need it.

Reply:
> Are you targeting Fedora server or Fedora CoreOS? I'm not sure which one Fedora cloud base refers to.

It is unclear as it feel to me Fedora cloud is another flavour. Do you think it can have a big impact?